client/python/samples/README.md

```diff
@@ -0,0 +1,223 @@
+# OpenVINO&trade; Model Server Client Library Example Clients
+
+This document contains examples to run *GetModelStatus*, *GetModelMetadata*, *Predict* functions over gRPC API.
+
+It covers following topics:
+* <a href="#grpc-api">gRPC API Example Clients </a>
+
+## Requirement
+
+**Note**: Provided examples and their dependencies are updated and validated for Python 3.6+ version. For older versions of Python, dependencies versions adjustment might be required.
+
+Start Model Server Client Library virtual environment
+```
+. ../lib/.venv/bin/activate
+```
+
+Install client dependencies using the command below in the example_client directory:
+```
+pip3 install -r requirements.txt
+```
```

Comment:
I believe it's not applicable for the client lib

Reply:
Oh, didn't catch it is for samples. The name of the directory could be misleading though.
Can we go just with:
`"Install client dependencies using the command below:"` 
?